commit d4e2ba2b4e42e401171c36d24b4368d58d3a1124
Author: Anthony Fieroni <bvbfan@abv.bg>
Date:   Sat Nov 18 08:24:26 2017 +0200

    Bump libraw minimal version + increase supported file extesions
    
    Differential Revision: https://phabricator.kde.org/D7335
    
    Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0934b9c..e0dee29 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,7 @@ message(STATUS "Starting CMake configuration for: ${PROJECT_NAME}")
 set(CMAKE_MIN_VERSION   "2.8.12")
 set(ECM_MIN_VERSION     "1.1.0")
 set(REQUIRED_QT_VERSION "5.2.0")
-set(LIBRAW_MIN_VERSION  "0.16")
+set(LIBRAW_MIN_VERSION  "0.18")
 
 cmake_minimum_required(VERSION ${CMAKE_MIN_VERSION})
 
diff --git a/src/rawfiles.h b/src/rawfiles.h
index 24ea979..1309fbd 100644
--- a/src/rawfiles.h
+++ b/src/rawfiles.h
@@ -27,6 +27,7 @@
 #define RAW_FILES_H
 
 // NOTE: extension list Version 1 and 2 are taken from http://www.cybercom.net/~dcoffin/dcraw/rawphoto.c
+// NOTE: extension list Version 5 is taken from https://github.com/MrKepzie/Natron/issues/408#issuecomment-84959815
 
 //                                           Ext    Descriptions From
 //                                                  www.file-extensions.org
@@ -82,18 +83,17 @@ static const char raw_file_extentions[] =
 
 // NOTE: VERSION 4
 
-                                          "*.srw "; // Samnsung Raw Image Format.
+                                          "*.srw " // Samnsung Raw Image Format.
 
-/* TODO: check if these format are supported
+// NOTE: VERSION 5
                                           "*.drf "  // Kodak Digital Camera Raw Image Format.
                                           "*.dsc "  // Kodak Digital Camera Raw Image Format.
                                           "*.ptx "  // Pentax Digital Camera Raw Image Format.
                                           "*.cap "  // Phase One Digital Camera Raw Image Format.
                                           "*.iiq "  // Phase One Digital Camera Raw Image Format.
-                                          "*.rwz "  // Rawzor Digital Camera Raw Image Format.
-*/
+                                          "*.rwz "; // Rawzor Digital Camera Raw Image Format.
 
 // increment this number whenever you change the above string
-static const int raw_file_extensions_version = 4;
+static const int raw_file_extensions_version = 5;
 
 #endif // RAW_FILES_H
